style/icomoon/demo.html

```diff
@@ -0,0 +1,70 @@
+<!doctype html>
```

Comment:
It isn't about a cost, it's about the fact that it will get deployed to your hosting and would end up publicly accessible using following path:
https://[easy-movie-search.surge.sh or another domain]/style/icomoon/demo.html